Code/GraphMol/CMakeLists.txt

```diff
@@ -9,8 +10,13 @@ rdkit_library(GraphMol
               Matrices.cpp Chirality.cpp RingInfo.cpp Conformer.cpp
               Renumber.cpp AdjustQuery.cpp Resonance.cpp StereoGroup.cpp
               new_canon.cpp SubstanceGroup.cpp
+
+              NewCIPLabelling/RDKitCipMol.cpp
```

Comment:
How about having this be a separate library instead of integrating it into GraphMol?

Reply:
That makes sense.

I didn't give too much thought to where to put it: I just added it to GraphMol because that was the fastest way I to be able use it, and it kind of made sense because that's where the other stereo stuff lives. But that was meant to be temporary, as well as the 'NewCIPLabelling' name...